### PR TITLE
Button: Repeat limitation for button content in label option's desc

### LINKED ISSUE #282: Button: Repeat limitations within option descriptions
The docs currently say:

> When using an input of type button, submit or reset, support is limited to plain text labels with no icons.

We should repeat this within the descriptions for the affected options.

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes #282

### CODE REVIEW

File: entries/button.xml [hunk 1]
```diff
@@ -49,7 +49,10 @@
 			<type name="Object" />
 		</option>
 		<option name="label" default="null" example-value='"custom label"'>
-			<desc>Text to show in the button. When not specified (<code>null</code>), the element's HTML content is used, or its <code>value</code> attribute if the element is an input element of type submit or reset, or the HTML content of the associated label element if the element is an input of type radio or checkbox.</desc>
+			<desc>
+				<p>Text to show in the button. When not specified (<code>null</code>), the element's HTML content is used, or its <code>value</code> attribute if the element is an input element of type submit or reset, or the HTML content of the associated label element if the element is an input of type radio or checkbox.</p>
+				<p>When using an input of type button, submit or reset, support is limited to plain text labels with no icons.</p>
```
Comment (scottgonzalez) [inline entries/button.xml]: Drop "with no icons" and add "When using an input of type button, submit or reset, icons are not supported." in the `icons` option.

**Comment by jzaefferer** [comment]
Updated.

**Comment by scottgonzalez** [comment]
Looks good.

**Comment by jzaefferer** [comment]
Landed in master. Also rebased the 1-12 branch on top, resolving the conflict from this change: https://github.com/jquery/api.jqueryui.com/pull/283/files#diff-bf14c9f5ccd16aa682040e25fcc9231eR49

**Pull request closed without merging**